.circleci/config.yml

```diff
@@ -865,10 +904,12 @@ jobs:
           name: Pull artifacts
           command: bash scripts/ops/pull-artifacts.sh
           working_directory: packages/contracts-bedrock
+      - go-restore-cache
       - run:
           name: Build go-ffi
           command: just build-go-ffi
           working_directory: packages/contracts-bedrock
+      - go-save-cache
```

Comment:
It feels odd to me that multiple different jobs are saving to the go cache. 

ie. why would the cache resulting from 
```
cd packages/contracts-bedrock
just build-go-ffi
``` 

be shared with the cache from 
```
cd op-acceptance-tests
go mod download
go build -o ../flake-shake-aggregator ./cmd/flake-shake-aggregator/main.go
```
?

Reply:
Maybe we should include the working_directory in the cache key?

Reply:
EEEeeeh I simplified a bit too much - I based this off another cache I was introducing in a different repo and thought to myself _why do I need this namespace I put here_ -  you are indeed completely right, I reintroduced the namespace/scoping for the cache